spec/models/user_model_spec.rb

```diff
@@ -47,6 +47,7 @@
 
   describe "ClassMethods" do
     before :all do
+      DatabaseCleaner.start
```

Comment:
Tried a few different ways that they recommended from the docs and none of them work with the test suite as it is written. I think that fully implementing this in the way that you want would require an overhaul of the test suite that is out of scope for this issue.